developer_manual/digging_deeper/dashboard.rst

```diff
@@ -0,0 +1,178 @@
+=========
+Dashboard
+=========
+
+The dashboard app aims to provide the user with a general overview of their
+Nextcloud and shows information that is currently important.
+
+App developers can integrate into the dashboard app and provide their own panels.
+
+
+Register a dashboard panels
+---------------------------
+
+A dashboard panel is represented by a class implementing the `OCP\\Dashboard\\IPanel`
+interface. This class will be instantiated whenever the dashboard is loaded.
+Any bootstrap code that is needed for the panel can be implemented inside
+of the `load` method and will be called when the dashboard is loaded.
+
+.. code-block:: php
+
+    <?php
+
+    namespace OCA\MyApp\Dashboard;
+
+    use OCP\Dashboard\IPanel;
+    use OCP\IInitialStateService;
+    use OCP\IL10N;
+    use OCP\IURLGenerator;
+    use OCP\IUserSession;
+
+    class MyAppPanel implements IPanel {
+
+        public function __construct(
+            IInitialStateService $initialStateService,
+            IL10N $l10n,
+            IURLGenerator $urlGenerator
+        ) {
+            $this->initialStateService = $initialStateService;
+            $this->l10n = $l10n;
+            $this->urlGenerator = $urlGenerator;
+        }
+
+        /**
+         * @return string Unique id that identifies the panel, e.g. the app id
+         * @since 20.0.0
+         */
+        public function getId(): string {
+            return 'myapppanelid';
+        }
+
+        /**
+         * @return string User facing title of the panel
+         * @since 20.0.0
+         */
+        public function getTitle(): string {
+            return $this->l10n->t('My app');
+        }
+
+        /**
+         * @return int Initial order for panel sorting
+         *   in the range of 10-100, 0-10 are reserved for shipped apps
+         * @since 20.0.0
+         */
+        public function getOrder(): int {
+            return 0;
+        }
+
+        /**
+         * @return string css class that displays an icon next to the panel title
+         * @since 20.0.0
+         */
+        public function getIconClass(): string {
+            return 'icon-class';
+        }
+
+        /**
+         * @return string|null The absolute url to the apps own view
+         * @since 20.0.0
+         */
+        public function getUrl(): ?string {
+            return $this->urlGenerator->linkToRouteAbsolute('myapp.view.index');
+        }
+
+        /**
+         * Execute panel bootstrap code like loading scripts and providing initial state
+         */
+        public function load(): void {
+            $initialStateService->provideInitialState('myapp', 'myData', []);
+            \OCP\Util::addScript('myapp', 'dashboard');
+        }
+    }
+
+
+The class needs to be registered during the app bootstrap.
```

Comment:
could you link to the bootstrapping page for those that read *app bootstrap* for the first time? There is a reference label `Bootstrapping` that you can use.